### PR TITLE
CM-55: adds bovine retrieve, delete and create

### PR DESCRIPTION
**Pull request by FranciscodeMaussion**

### CODE REVIEW

**Comment by FranciscodeMaussion** [comment]
test this please

File: bovine-be.iml [hunk 1]
```diff
@@ -0,0 +1,153 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (merlindk) [inline bovine-be.iml]: This file should not be included. Verify that .iml is in the .gitignore

File: src/main/java/com/cema/bovine/controllers/Controller.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    })
+    @PostMapping(value = base_url, produces = {MediaType.APPLICATION_JSON_VALUE}, consumes = {MediaType.APPLICATION_JSON_VALUE})
+    public ResponseEntity<Bovine> registerBovine(
+            @ApiParam(
+                    value = "Bovine data to be inserted.")
+            @RequestBody Bovine bovine) {
+
+        LOG.info("Request to register new bovine");
+
+        CemaBovine exists_bovine = bovineRepository.findCemaBovineByTag(bovine.getTag());
```
Comment (merlindk) [inline src/main/java/com/cema/bovine/controllers/Controller.java]: can we keep it camel case?

File: src/main/resources/application.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+info:
```
Comment (merlindk) [inline src/main/resources/application.yml:1]: Please add environments qa and prod

File: src/main/java/com/cema/bovine/controllers/Controller.java (hunk 2 above)
Comment (FranciscodeMaussion) [reply src/main/java/com/cema/bovine/controllers/Controller.java]: :facepalm:

File: src/main/resources/application.yml (hunk 3 above)
Comment (FranciscodeMaussion) [reply src/main/resources/application.yml:1]: done

File: bovine-be.iml (hunk 1 above)
Comment (FranciscodeMaussion) [reply bovine-be.iml]: kk

**Comment by merlindk** [comment]
test this please

**Comment by merlindk** [comment]
test this please

**Review by merlindk — APPROVED** [review]
LGTM

**Comment by FranciscodeMaussion** [comment]
test this please

**Pull request merged**